src/components/SwatchPaletteWidget/_swatch-palette-widget.scss

```
@@ -90,3 +90,7 @@
     height: 100%;
   }
 }
+
+[class*='--col'].#{$prefix}--swatch-palettes > * + * {
```

If a 'stack' doesn't have different types of components, there's no need for the loose selector, we can just define `--space` with the child.

```suggestion
.#{$prefix}--swatch-palette {
```